package.json

```diff
@@ -14,17 +14,17 @@
     "cookie-parser": "1.4.3",
     "express": "4.14.0",
     "express-session": "1.14.2",
-    "infusion": "2.0.0-dev.20161027T155641Z.0bcd507",
-    "resolve": "1.1.7"
+    "infusion": "2.0.0-dev.20161219T170555Z.5778f7e",
+    "resolve": "1.2.0"
```

Comment:
As far as I can see, this dependency is not used within gpii-express. Also, if it were, the version you have nominated (and all currently released versions of "resolve") suffer from https://github.com/substack/node-resolve/issues/106 and should be replaced in the interim by "fluid-resolve".